Assets/_MUTUAL/Installers/MainSceneInstallers.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Mutual.Installers;
+using Mutual.Managers;
+using UnityEngine;
+using Zenject;
+
+namespace Mutual.Installers
+{
+    public class MainSceneInstallers : MonoInstaller
+    {
+        private AppManager m_AppManager;
+
+        public override void InstallBindings()
+        {
+            // Non MonoInstallers
+
+            InstallDependencies();
+
+            // Mono behaviors 
+
+            m_AppManager = GameObject.FindObjectOfType<AppManager>();
```

Comment:
@RemyaTP Not the worst thing to use here, but in case you're interested, GameObject.FindObjectOfType is a **very expensive** call

Reply:
@KICO-SOURCE Replaced with GetComponent, is that ok?

Reply:
Yep, that's all good.